site/components/src/main/java/uk/nhs/hee/web/components/BulletinComponent.java

```diff
@@ -0,0 +1,170 @@
+package uk.nhs.hee.web.components;
+
+import com.google.common.base.Strings;
+import org.hippoecm.hst.container.RequestContextProvider;
+import org.hippoecm.hst.content.beans.query.HstQuery;
+import org.hippoecm.hst.content.beans.query.HstQueryResult;
+import org.hippoecm.hst.content.beans.query.builder.HstQueryBuilder;
+import org.hippoecm.hst.content.beans.query.exceptions.FilterException;
+import org.hippoecm.hst.content.beans.query.exceptions.QueryException;
+import org.hippoecm.hst.content.beans.query.filter.Filter;
+import org.hippoecm.hst.content.beans.standard.HippoBean;
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.component.HstResponse;
+import org.hippoecm.hst.core.parameters.ParametersInfo;
+import org.onehippo.cms7.essentials.components.CommonComponent;
+import org.onehippo.cms7.essentials.components.paging.Pageable;
+import org.onehippo.cms7.essentials.components.utils.SiteUtils;
+import org.onehippo.forge.selection.hst.contentbean.ValueList;
+import org.onehippo.forge.selection.hst.util.SelectionUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.nhs.hee.web.components.info.BulletinComponentInfo;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+@ParametersInfo(type = BulletinComponentInfo.class)
+public class BulletinComponent extends CommonComponent {
+    private static final Logger log = LoggerFactory.getLogger(BulletinComponent.class);
+
+    private final static String CATEGORY_QUERY_PARAM = "category";
+    private final static String CATEGORY_BEAN_ATTRIBUTE = "hee:category";
```

Comment:
Should we put this constant into a separated class like DocumentPropertyNames to reuse it later on other cases ?